.github/workflows/ci.yml

```diff
@@ -323,17 +347,42 @@ jobs:
       version-set: ${{ toJson(matrix.version-set) }}
     secrets: inherit
 
-  # Tests that depend on builds, but a smaller subset against Windows & MacOS platforms.
-  acceptance-test:
+  # Tests that depend on builds, but a smaller subset against Windows platform.
+  acceptance-test-win:
```

Comment:
There are a couple `needs:` lines lower down in the file that are referencing `acceptance-test` that will need to be updated for the new job names.